client/src/components/Routine-Manager/FocusedRoutine.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import {
+  postNewExerciseInRoutine,
+  updateExercise,
+  updateRoutine,
+  scheduleWorkout
+} from "../../actions";
+
+import "../../less/workouts.css";
+
+class FocusedRoutine extends Component {
+  render() {
+    return (
+      <div className="RoutineBuilder">
+        <div className="RoutineBuilder__Title">
+          {this.props.selectedRoutine ? (
+            <input
+              name="RoutineName"
+              placeholder={this.props.selectedRoutine.title}
+            />
+          ) : (
+            <div>Loading Content...</div>
+          )}
+        </div>
+        <div className="RoutineBuilder__Tags" />
+        <button
+          className="RoutineBuilder__AddExercise"
+          onClick={() =>
+            this.props.postNewExerciseInRoutine(this.props.selectedRoutine._id)
+          }
+        >
+          Add an exercise
+        </button>
+        {this.props.selectedRoutine ? (
+          this.props.selectedRoutine.exercises.map(
+            (exerciseInRoutine, index) => {
+              let {
+                name,
+                currentReps,
+                currentSets,
+                currentWeight
+              } = exerciseInRoutine;
+              return (
+                <form key={index} className="RoutineBuilder__ExerciseCard">
+                  <input
+                    name="ExerciseName"
+                    onChange={event => (name = event.target.value)}
+                    placeholder={exerciseInRoutine.name}
+                    className="RoutineBuilder__ExerciseCardField--name"
+                  />
+                  <input
```

Comment:
For the next iteration, I would suggest putting a `type` in your weight, sets and reps so that nothing other than numbers can be input. Also, this may be in for when you do styling but putting "Weight" "Sets" and "Reps" so that users will know what the boxes are for.